docs/webadministration.rst

```diff
@@ -0,0 +1,436 @@
+The default value is 80%. The lower the spam filter tolerance, the more false positives (ham classified as spam). The higher the spam filter tolerance, the more false negatives (spam classified as ham). 
+For more information see the :ref:`antispam documentation <antispam_howto>`.
+
+Auto-forward
+`````````````
+Under the section `Auto-forward`, the automatic forwarding of received email messages can be enabled. When enabled, all received email messages are forwarded to the specified email address. 
+
+The option "Keep a copy of the emails" can be ticked, to keep a copy of the received email message in the inbox folder.
+
+In the destination textbox, the email addresses can be entered for automatic forwarding. When entering multiple email addresses the comma (',') must be used as delimiter.
+
+
+Update password
+---------------
+
+On the `update password` page, the password of the logged in user can be changed. Changes are effective immediately. 
+
+
+.. _webadministration_auto-reply:
+
+Auto-reply
+----------
+
+On the `auto-reply` page, automatic replies can be configured. This is also known as out of office replies.
+
+To enable automatic replies tick the checkbox 'Enable automatic reply'. 
+
+Under Reply subject the email subject for automatic replies can be configured. When a reply subject is entered, this subject will be used for the automatic reply.
+
+When no reply subject is entered, the automatic reply will have the subject `auto: <subject from received email>`.
+E.g. if the email subject of the received email message is "how are you?", then the email subject of the automatic reply is `auto: how are you?`.
+
+
+.. _webadministration_fetched_accounts:
+
+Fetched accounts
+----------------
+
+This page is only available when the Fetchmail container is part of your Mailu deployment. 
+Fetchmail can be enabled when creating the docker-compose.yml file with the setup utility (https://setup.mailu.io).
+
+On the `fetched accounts` page you can configure email accounts from which email messages will be retrieved.
+Only unread email messages are retrieved from the specified email account. 
+By default Fetchmail will retrieve email messages every 10 minutes. This can be changed in the Mailu.env file. 
+For more information on changing the polling interval see :ref:`the configuration reference <fetchmail>`.
+
+
+You can add a fetched account by clicking on the `Add an account` button on the top right of the page. To add an fetched account, the following settings must be configured:
+
+* Protocol (IMAP or POP3). The protocol used for accessing the email server.
+
+* Hostname or IP. The hostname or IP address of the email server.
+
+* TCP port. The TCP port the email server listens on. Common ports are 993 (TLS) or 143 (STARTTLS/unsecure) for IMAP. 
+
+* Enable TLS. Tick this setting if the email server requires TLS/SSL.
```

Comment:
```suggestion
* Enable TLS. Tick this setting if the email server requires TLS/SSL instead of STARTTLS.
```

imo this makes it more clearer, that encryption is used in any case.

Reply:
Also should we use the term SSL? Many people are probably still searching for it, but technically it's deprecated and should just vanish some day. The interface itself also just says TLS, as noted in the beginning of the line.

Reply:
for now lets stick to SSL. For many people SSL is the same as TLS. We can always remove this in the future once SSL is not commonly used anymore to refer to a secure connection.